keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -204,7 +204,7 @@ Add ability to support drop-in configuration directory.
 Add ability to augment the feature's capabilities with a focus on robustness and testing, which includes:
   - Ensure the correct kubelet configuration is displayed when queried using the `kubectl get --raw "/api/v1/nodes/{node-name}/proxy/configz"` command, particularly verifying the contents of the kubelet.conf.d directory.
   - Remove the environment variable `KUBELET_CONFIG_DROPIN_DIR_ALPHA`, introduced during the Alpha phase, to streamline the user experience by simplifying configuration management.
-  - Set the default value of `--config-dir` to `/etc/kubernetes/kubelet.conf.d`. Users can disable this field by setting it to an empty string.
+  - Leave the `--config-dir` flag empty by default. Users can configure it by specifying a path.
```

Comment:
Can we add the "recommended" path to the argument description? This way we may achieve higher portability of third party configuration tooling.

Reply:
other than that, lgtm